extensions/douban/src/search-games.tsx

```diff
@@ -99,6 +100,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${game.title}(豆瓣)](${game.url})`}
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.